sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorLogRecordExporter.java

```diff
@@ -54,6 +55,9 @@ public CompletableResultCode export(Collection<LogRecordData> logs) {
 
         List<TelemetryItem> telemetryItems = new ArrayList<>();
         for (LogRecordData log : logs) {
+            if (log.getInstrumentationScopeInfo().getName().startsWith(LOGGER_PREFIX)) {
+                continue;
+            }
```

Comment:
it may be safer to use a thread local to prevent exporting our logs, e.g. to cover cases where our exporter calls azure sdk core classes which do their own verbose logging

Reply:
let's keep as-is and in the future (once released) we can use https://github.com/open-telemetry/opentelemetry-java/pull/6546

Reply:
i added a todo for reminder